src/main/resources/application.yaml

```diff
@@ -13,6 +13,7 @@ management:
 
 springdoc:
   packagesToScan: uk.gov.hmcts.reform.demo.controllers
+  writer-with-order-by-keys: true
```

Comment:
Makes the API spec deterministic so that the `git diff` returns no changes so we don't publish a new version on every commit to master